packages/scripts/src/upgrade-tests/voting2/2_Verify.ts

```diff
@@ -127,6 +133,22 @@ async function main() {
   assert((await votingToken.balanceOf(governor.address)).eq(0));
   console.log("✅ Governor v2 received all the voting tokens from Governor!");
 
+  if (await isGovernorV2Instance(governorV2.address)) {
+    const emergencyProposerAddress = process.env["EMERGENCY_PROPOSER_ADDRESS"];
+    const emergencyProposer = await getContractInstance<EmergencyProposerEthers>(
+      "EmergencyProposer",
+      emergencyProposerAddress
+    );
+    if (!emergencyProposerAddress) throw new Error("Missing EMERGENCY_PROPOSER_ADDRESS env variable");
+    console.log(" 11. Governor v2 is the owner of the EmergencyProposer...");
+    assert.equal((await emergencyProposer.owner()).toLowerCase(), governorV2.address.toLowerCase());
+    console.log("✅ EmergencyProposer owner role correctly set!");
+
+    console.log(" 12. EmergencyProposer has the emergency proposer role in GovernorV2 ...");
+    assert.equal((await governorV2.getMember(2)).toLowerCase(), emergencyProposer.address.toLowerCase());
+    console.log("✅ EmergencyProposer has the emergency proposer role in GovernorV2!");
+  }
+
```

Comment:
Should also check that `executor` address is correctly set in the `EmergencyProposer`

Reply:
Nice catch. Thanks.

Fixed in the latest commit.